modules/dxp/apps/saml/saml-persistence-service/src/main/java/com/liferay/saml/persistence/service/persistence/impl/SamlPeerBindingPersistenceImpl.java

```diff
@@ -0,0 +1,1748 @@
+/**
```

Comment:
When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation: